docs/features/performance_view.md

```diff
@@ -13,9 +13,9 @@ Specifications:
   - 16 FX columns
     - 1 Param assigned to each column
   - 8 FX values per column
-    - Long press pads in a column to change value momentarily and reset it (to the value before the pad was pressed) upon pad release
-    - Short press pads in a column to the change value until you press the pad again (resetting it to the value before the pad was pressed)
-    - Quickly clear all held values using `HORIZONTAL ENCODER ◀︎▶︎` + `BACK` (resetting FX values back to their previous state)
+    - Long press pads (>= 0.5 sec) in a column to change value momentarily and reset it (to the value before the pad was pressed) upon pad release
```

Comment:
the exact momentary vs. latch times are configurable in the global settings menu, but I believe the default is 0.2 seconds. That value controls all spots where taps latch and holds don't (sticky shift, keyboard sidebar, and performance view)

0.5s is used to distinguish short vs. long press. That's the value used for save vs. collect all save, load vs cloning synths, and selecting vs. entering clips from song view (probably more). 

It became clear to us during the beta that those actions needed very different definitions of how long to press to make them feel natural

Reply:
On the "more" for 0.5seconds, it's also used for selecting and removing notes in the instrument clip views.

Reply:
Good to know! Should be resolved now